finality-aleph/src/metrics/best_block_related.rs

```diff
@@ -0,0 +1,177 @@
+use substrate_prometheus_endpoint::{
+    register, Gauge, Histogram, HistogramOpts, PrometheusError, Registry, U64,
+};
+
+use crate::{block::TreePathAnalyzer, BlockId};
+
+pub enum BestBlockRelatedMetrics<TPA> {
```

Comment:
Nit: The "related" in this name is kinda unnecessary, especially with "metrics" being plural.

Reply:
maybe we can change the filename too

Reply:
Renamed to `best_block`

Reply:
Even nitter: Missed the names in `metrics/slo.rs`.

Reply:
Good spotting

Reply:
Ahh, you really like the "related", don't you? xd You still left one variable with that in the constructor and then you have to use the `foo: bar` notation. I _hope_ this was the result of you using some automated renaming, not actually trying to keep it. :P

Reply:
It seems that my IDE liked the previous name more. Fixed and hope my IDE won't eat my whole ram next time as a revenge.